models/istio_config.go

```diff
@@ -49,9 +48,42 @@ type IstioConfigDetails struct {
 	PeerAuthentication    *security_v1beta.PeerAuthentication    `json:"peerAuthentication"`
 	RequestAuthentication *security_v1beta.RequestAuthentication `json:"requestAuthentication"`
 
-	Permissions     ResourcePermissions `json:"permissions"`
-	IstioValidation *IstioValidation    `json:"validation"`
-	IstioReferences *IstioReferences    `json:"references"`
+	Permissions           ResourcePermissions `json:"permissions"`
+	IstioValidation       *IstioValidation    `json:"validation"`
+	IstioReferences       *IstioReferences    `json:"references"`
+	IstioConfigHelpFields []IstioConfigHelp   `json:"help"`
+}
+
+// IstioConfigHelp represents a help message for a given Istio object type and field
+type IstioConfigHelp struct {
+	ObjectField string `json:"objectField"`
+	Message     string `json:"message"`
+}
+
+var IstioConfigHelpMessages = map[string][]IstioConfigHelp{
+	"virtualservices": {
+		{ObjectField: "spec.hosts", Message: "The destination hosts to which traffic is being sent. Could be a DNS name with wildcard prefix or an IP address. Depending on the platform, short-names can also be used instead of a FQDN (i.e. has no dots in the name)."},
```

Comment:
Is this duplicated with the https://github.com/kiali/kiali/pull/4740/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R724 ?

Reply:
Yes, sorry. I removed the old code. Thanks @lucasponce .